src/tests/index.test.js

```diff
@@ -112,7 +112,7 @@ async function validateClient(evolv, options, uid, sid) {
   evolv.confirm();
 
   evolv.context.set('user_attributes.country', 'usa');
-  expect(contextChangedSpy).to.have.been.called(6);
+  expect(contextChangedSpy).to.have.been.called;
```

Comment:
This assertion becomes meaningless if you take away the call count. Since the assertion was true on line 97, it will continue to be true on all subsequent lines of the test.

Has it become unmaintainable to discern how many times the spy has been called with every code change?

Reply:
From my view yes. It keeps changing.